modules/dxp/apps/scim/scim-rest-impl/rest-openapi.yaml

```diff
@@ -38,6 +38,16 @@ components:
                               $ref: "#/components/schemas/MultiValuedAttribute"
                           type: array
                   type: object
+        LiferayUserSchemaExtension:
+            description:
+                Liferay's User Schema Extension.
+            properties:
+                birthday:
+                    format: date-time
+                    type: string
```

Comment:
Hey @daiazius, use the `date` format instead as it represents a birthday. I'm not sure if it will change the expected format but it is semantically correct.

Thx